client/my-sites/importer/uploading-pane.jsx

```diff
@@ -70,6 +86,9 @@ class UploadingPane extends React.PureComponent {
 		switch ( importerState ) {
 			case appStates.READY_FOR_UPLOAD:
 			case appStates.UPLOAD_FAILURE:
+				if ( this.state.fileToBeUploaded ) {
+					return <p>{ this.props.translate( 'Click Upload to begin uploading the file' ) }</p>;
```

Comment:
When referencing other UI it's favorable to use the translation as variable (to ensure it really says the same text on the button).
But actually I was wondering if we could display selected filename instead? Personally I like to be able to confirm that I have selected the right file before I submit the form.

Reply:
The UI looks like this now. I agree it is much more useful to see the file name. Thank you!

<img width="625" alt="CleanShot 2021-07-02 at 12 08 41@2x" src="https://user-images.githubusercontent.com/533/124231709-48207f80-db2e-11eb-8f27-01f7e035c947.png">

The clicking Upload should be obvious to a user, but maybe if it isn't, we can add an additional message once we do some user testing.